week-1/G-strings-methods/exercise2.js

```diff
@@ -1,3 +1,2 @@
-var name = " Daniel  ";
-
-console.log(message);
+var message = "   My name is Saeed and my name is 5 characters long   ";
```

Comment:
This exercise asks you to use the given variable and the `trim()` method to print out the same message as the previous one.